cmd/libsnap-confine-private/infofile.c

```diff
@@ -28,6 +28,10 @@
 #include "../libsnap-confine-private/utils.h"
 
 int sc_infofile_get_key(FILE *stream, const char *key, char **value, sc_error **err_out) {
+    return sc_infofile_get_ini_key(stream, NULL, key, value, err_out);
+}
+
+int sc_infofile_get_ini_key(FILE *stream, const char *section, const char *key, char **value, sc_error **err_out) {
```

Comment:
section_key?

Reply:
Now I see that the comment can be confusing. What I means is maybe the function should be named `sc_infofile_get_ini_section_key`

Reply:
Renamed.